src/frontend/src/pages/ProjectPage/pages/ContractPage/pages/ManagePersonnelPage/index.tsx

```diff
@@ -35,16 +35,14 @@ const ManagePersonnelPage: React.FC = () => {
             return;
         }
 
-        try {
-            const result = await apiClient.getPersonnelWithPositionsAsync(projectId, contractId);
-            dispatchContractAction({
-                verb: 'merge',
-                collection: 'personnel',
-                payload: result,
-            });
-        } catch (error) {
-            return error;
-        }
+        const result = await apiClient.getPersonnelWithPositionsAsync(projectId, contractId);
+        dispatchContractAction({
+            verb: 'merge',
+            collection: 'personnel',
+            payload: result,
+        });
+
+        return error;
```

Comment:
Where is error defined?

Reply:
was part of the try catch, Ive removed it now.